### PR TITLE
JP : HD resolution + more channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
thx to @luongz for additional research.

### CODE REVIEW

**Review by luongz — DISMISSED** [review]

File: streams/jp.m3u [hunk 1]
```diff
@@ -55,8 +57,12 @@ http://51.79.193.108:8080/hls/j00039/index.m3u8?type=jp
 #EXTINF:-1 tvg-id="JSports4.jp" status="error",J Sports 4 (720p) [Not 24/7]
 http://51.79.193.108:8080/hlsok/j00039/index.m3u8?type=jp
 #EXTINF:-1 tvg-id="KidsStation.jp" status="online",Kids Station
+http://cdn.us195.jpnettv.live:1935/jptv/jkids_720/playlist.m3u8
```
Comment (BountyChocolate123456) [inline streams/jp.m3u:60]: offline?

File: streams/jp.m3u [hunk 2]
```diff
@@ -55,8 +57,12 @@ http://51.79.193.108:8080/hls/j00039/index.m3u8?type=jp
...
+#EXTINF:-1 tvg-id="KidsStation.jp" status="online",Kids Station
 http://cdn.us195.jpnettv.live:1935/jptv/jkids_540/playlist.m3u8
 #EXTINF:-1 tvg-id="MoviePlus.jp" status="online",Movie +
+http://cdn.us195.jpnettv.live:1935/jptv/movieplus_720/playlist.m3u8
```
Comment (BountyChocolate123456) [inline streams/jp.m3u:64]: offline?

File: streams/jp.m3u [hunk 3]
```diff
@@ -106,22 +114,16 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel
 https://movie.mcas.jp/mcas/mx1_2/master.m3u8
 #EXTINF:-1 tvg-id="TokyoMX1.jp" status="online",Tokyo MX1 (720p)
 https://movie.mcas.jp/mcas/mx_live_2/master.m3u8
-#EXTINF:-1 tvg-id="TokyoMX1.jp" status="online",Tokyo MX 1
+#EXTINF:-1 tvg-id="TokyoMX1.jp" status="online",Tokyo MX1
+http://cdn.us195.jpnettv.live:1935/jptv/kanto_tokyo_mx1_720/playlist.m3u8
```
Comment (BountyChocolate123456) [inline streams/jp.m3u:118]: offline?

File: streams/jp.m3u [hunk 4]
```diff
@@ -160,30 +172,74 @@ http://cdn.us195.jpnettv.live:1935/jptv/Fishing_broadcasting/playlist.m3u8
...
-#EXTINF:-1 tvg-id="" status="error",Kansai TV (720p) [Not 24/7]
-http://51.79.193.108:8080/hlsok/j00006/index.m3u8?type=jp
 #EXTINF:-1 tvg-id="" status="error",GAORA (540p) [Not 24/7]
 http://51.79.193.108:8080/hlsok/j00045/index.m3u8?type=jp
+#EXTINF:-1 tvg-id="NHKGeneralTV.jp",NHK G Osaka
+http://cdn.us195.jpnettv.live:1935/jptv/kansi_nhk_g_720/playlist.m3u8
+#EXTINF:-1 tvg-id="NHKGeneralTV.jp",NHK G Osaka
+http://cdn.us195.jpnettv.live:1935/jptv/kansi_nhk_g_540/playlist.m3u8
+#EXTINF:-1 tvg-id="NHKEducationalTV.jp",NHK E Osaka
+http://cdn.us195.jpnettv.live:1935/jptv/kansi_nhk_edu_720/playlist.m3u8
```
Comment (BountyChocolate123456) [inline streams/jp.m3u:207]: offline?

**Comment by luongz** [comment]
links down
xd

File: streams/jp.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply streams/jp.m3u:118]: works

Comment (LaneSh4d0w) [reply streams/jp.m3u:118]: snapshot

**Review by freearhey — APPROVED** [review]

**Review by niyazijsjj — APPROVED** [review]

**Review by Central302 — APPROVED** [review]

**Pull request merged**